pages/1.12/cli/command-reference/dcos-security/dcos-security-cluster/dcos-security-cluster-directory-import-group/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: layout.pug
+navigationTitle:  dcos security cluster directory import_group
+title: dcos security cluster directory import_group
+menuWeight: 33
+excerpt: Importing an LDAP group
+enterprise: true
+---
+
+# Description
+
+The `dcos security cluster directory import_group` command imports a group of users from the configured directory (LDAP) backend. See IAM documentation for details on group import.
```

Comment:
Last sentence can link to `/1.12/security/ent/users-groups/add-external-user/#importing-groups-of-ldap-users`

Reply:
Will do. Thanks!